README.md

```diff
@@ -117,3 +116,41 @@ is "friendlier". If you have docker installed, use [this
 script](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/scripts/run_console_local.sh)
 to fire it up on <http://localhost:9000>.
 
+#### Using kubectl
+
+To install Knative Eventing into the `knative-eventing` namespace, apply
+the following resources:
+
+```
+cat <<-EOF | kubectl apply -f -
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: knative-eventing
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: knative-eventing-operator-sub
+  generateName: knative-eventing-operator-
+  namespace: knative-eventing
+spec:
+  source: knative-eventing-operator
+  sourceNamespace: olm
+  name: knative-eventing-operator
+  channel: alpha
+---
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Install
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+EOF
```

Comment:
nice addition!

Reply:
thanks! that can be copied to [the README in the fork](https://github.com/openshift/knative-eventing/blob/release-v0.5.0/openshift/olm/README.md), too, in case anyone ever looks at that.t